### PR TITLE
Added dev environment for Fabric

### PR DESCRIPTION
**Pull request by litong01**

### CODE REVIEW

File: docs/dev-environments/fabfic/README.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+### Extract Avalon [chaincode](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/fabric/chaincode) and deploy them on the Fabirc network
+
+```
+./getandinstall.sh
+```
+
+
+### Run your tests
+Once you have all the chaincode installed and connection profile avalable, you can use [Avalon python connector libary](https://github.com/hyperledger/avalon/tree/master/sdk/avalon_sdk/fabric) to make Avalon API 1.1 calls.
```
[the remark this answers is not in the document]
Comment (danintel) [reply docs/dev-environments/fabfic/README.md]: Also change the directory name from `fabfic` to `fabric`

File: docs/dev-environments/fabfic/README.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Testing Avalon with a Hyperledger Fabric blockchain
```
Comment (danintel) [inline docs/dev-environments/fabfic/README.md]: Add a CCL license. For example:
```
<!--
Licensed under Creative Commons Attribution 4.0 International License
https://creativecommons.org/licenses/by/4.0/
-->

Also change directory from `fabfic` to `fabric`
```

Comment (litong01) [reply docs/dev-environments/fabfic/README.md]: ok. sorry for the obvious error.

File: docs/dev-environments/fabfic/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (litong01) [reply docs/dev-environments/fabfic/README.md]: ok

File: shared_kv_storage/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,98 @@
+# Copyright 2019 Intel Corporation
```
Comment (Ram-srini) [inline shared_kv_storage/Dockerfile]: Why your PR has changes to shared_kv_storage module? I think you need to re-base your PR.

**Comment by Ram-srini** [comment]
Please add commit description, it is empty now.

File: ci/docker-compose-build.yaml [hunk 4]
```diff
@@ -29,7 +29,7 @@ services:
     image: avalon-lmdb:${ISOLATION_ID}
     build:
       context: ..
-      dockerfile: examples/shared_kv_storage/Dockerfile
+      dockerfile: shared_kv_storage/Dockerfile
```
Comment (manojgop) [inline ci/docker-compose-build.yaml]: This change is already present in latest code in master branch. Looks like you might need to rebase.

Comment (litong01) [reply ci/docker-compose-build.yaml]: this is very strange, I rebased but it still shows this.

Comment (pankajgoyal2) [reply ci/docker-compose-build.yaml]: The commits in the PR shows other commits as well. Probably, you included those commits while rebasing. that's why you see the above changes in this PR.

Comment (litong01) [reply ci/docker-compose-build.yaml]: The commit on my own remote repo shows just one commit. But here shows 3. Is there an easy way to fix this not losing all the comments of this PR? I mean not closing this PR.

**Comment by litong01** [comment]
I will do a new PR. This becomes way to complicated.

**Pull request closed without merging**